### DISCUSSION TITLE
Cannot update: "Illuminate\Contracts\Container\BindingResolutionException"

### DISCUSSION THREAD

**Discussion by gittbuh** [firefly-iii/firefly-iii#8734] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

I have tried to update my self-managed firefly instance following the now recommended "Straight from GitHub" option following to the letter the instructions here https://docs.firefly-iii.org/how-to/firefly-iii/upgrade/self-managed/ and the thing will not upgrade, it gets stuck at the first php artisan command, throwing an "Illuminate\Contracts\Container\BindingResolutionException" error message. 

My firefly instance has been running without issues in a Proxmox LXC container for a few years now and went already through several updates but with the old composer create-project mode, never had issues before. I am updating from version 6.1.0. 

### Debug information

 Illuminate\Contracts\Container\BindingResolutionException 

  Target class [hash] does not exist.

  at vendor/laravel/framework/src/Illuminate/Container/Container.php:906
    902▕ 
    903▕         try {
    904▕             $reflector = new ReflectionClass($concrete);
    905▕         } catch (ReflectionException $e) {
  ➜ 906▕             throw new BindingResolutionException("Target class [$concrete] does not exist.", 0, $e);
    907▕         }
    908▕ 
    909▕         // If the type is not instantiable, the developer is attempting to resolve
    910▕         // an abstract type such as an Interface or Abstract Class and there is

      +13 vendor frames 

  14  app/Exceptions/Handler.php:198
      Illuminate\Auth\AuthManager::__call()
      +3 vendor frames 

  18  artisan:35
      Illuminate\Foundation\Console\Kernel::handle()

**Comment by JC5** [comment]
A few closed issues refer to a possible solution: run `composer install` in your Firefly III directory.
